### DISCUSSION TITLE
alias path name

### DISCUSSION THREAD

**Discussion by BhaskaranR** [hannoeru/vite-plugin-pages#138] [General]
is there anyway i could alias path name for a dir?

```
{ dir: 'src/features/**/pages', baseRoute: 'features' , alias: 'milkrun'},
```

**Comment by BhaskaranR** [comment]
nvm got this to work. great job building this plugin.

**Reply by michaelpumo** [reply]
You should probably post the answer for other people.

**Comment by LikitSaeLee** [comment]
was searching for this answer, apparently you can use the SFC config.

```
<route>
{
  "alias": ["/path/one", "/path/two"]
}
</route>
```
